### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by vincemtnz**
Auth0 authentication as an example (does not persist to local or session storage as it's just a reference implementation of actionCreators and reducers).

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -29,5 +30,6 @@
       s.parentNode.insertBefore(wf, s);
     })();
   </script>
+  <script src="https://cdn.auth0.com/js/lock-8.min.js"></script>
```
Comment (mcalthrop) [inline src/index.html:33]: Can we include this as a lib?

Comment (vincemtnz) [inline src/index.html:33]: Not without adding a few other build-pipeline dependencies (and this authentication approach is not the only one you'd ever use so it'd just populate the dependencies list with modules that we don't use).

File: src/pages/NotFoundView/NotFoundView.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Link} from 'react-router';
+
+export class NotFoundView extends React.Component {
```
Comment (mcalthrop) [inline src/pages/NotFoundView/NotFoundView.js]: Why does this class have the suffix `View`, when other classes in the `pages` directory have suffix `Page`?

File: src/index.html (hunk 1 above)
Comment (vincemtnz) [inline src/index.html:33]: More info here: https://github.com/auth0/lock#webpack

File: src/pages/ProfileEditPage/ProfileEditPage.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+export class NotFoundView extends React.Component {
```
Comment (mcalthrop) [inline src/pages/ProfileEditPage/ProfileEditPage.js]: Should this class name be `ProfileEditPage`?

Comment (vincemtnz) [inline src/pages/ProfileEditPage/ProfileEditPage.js]: Copypasta fail.

**Pull request merged**